tests/by-util/test_split.rs

```diff
@@ -659,6 +659,22 @@ fn test_line_bytes_no_empty_file() {
     assert!(!at.plus("xak").exists());
 }
 
+#[test]
```

Comment:
I am confused why this isn't testing the "-io" option too ?

Reply:
The `---io` option is unspecified behavior for GNU split. The code will currently accept and ignore any ---io flags. It is not clear how the flag should be handled as the "chunking" of data is very different.